src/System/Status/StatusChecker.php

```diff
@@ -220,6 +220,10 @@ public static function getDBStatus($public_only = true): array
             }
         }
 
+        // Set new properties. Master and slave are deprecated given their implications in English.
+        $status['main'] = $status['master'];
```

Comment:
I wonder if this would not be preferable to really rename all occurrences, and potentially keep a master/slave here for backward compatibility.